dockers/base/Dockerfile

```diff
@@ -36,13 +39,8 @@ RUN apt-get update && apt-get install -y --no-install-recommends \
     && apt-get clean \
     && rm -rf /var/lib/apt/lists/*
 
-WORKDIR /opt
-COPY versions/GO_VERSION .
-RUN export GO_VERSION="$(cat GO_VERSION)" \
-    && curl -sSL -O "https://dl.google.com/go/go${GO_VERSION}.linux-amd64.tar.gz" \
-    && tar zxf "go${GO_VERSION}.linux-amd64.tar.gz" \
-    && rm "go${GO_VERSION}.linux-amd64.tar.gz" \
-    && ln -s /opt/go/bin/go /usr/bin/ \
+COPY --from=golang /usr/local/go /opt/go
+RUN ln -s /opt/go/bin/go /usr/bin/ \
```

Comment:
$GOROOT/bin includes multiple binary. it would be better to add $GOROOT/bin to $PATH

Reply:
thanks. fixed